scripts/Diffraction/isis_powder/routines/common.py

```diff
@@ -156,7 +157,10 @@ def get_first_run_number(run_number_string):
     :return: The first run for the user input of runs
     """
     run_numbers = generate_run_numbers(run_number_string=run_number_string)
-    return run_numbers[0]
+    if isinstance(run_numbers, list):
```

Comment:
Thats correct - it means the caller doesn't have to figure out types on their end (e.g. can just use a for in loop or [0])

Good spot 👍